Resources/Prototypes/Nyanotrasen/Entities/Objects/Weapons/Melee/breaching_hammer.yml

```
@@ -12,6 +12,7 @@
   - type: MeleeWeapon
     attackRate: 0.4
     range: 1.70
+    wideAnimationRotation: -135 # DeltaV: Added wideAnimationRotation
```

Also no change comment needed in Nyanotrasen, that's just dV but old
```suggestion
    wideAnimationRotation: -135
```